README.md

```diff
@@ -62,14 +65,14 @@ To form a cluster from scratch, start a few instances of the Crate container as
 daemon:
 
 ```console
-# docker run -d crate/crate
+# docker run -d crate/crate crate
 ```
 
 To access the admin UI, map port 4200 and point your browser to port tcp/4200 of
 a node of your choice while you start it or look up its IP later:
 
 ```console
-# firefox "http://$(docker inspect --format='{{.NetworkSettings.IPAddress}}' $(docker run -d crate/crate)):4200/admin"
+# firefox "http://$(docker inspect --format='{{.NetworkSettings.IPAddress}}' $(docker run -d crate/crate crate)):4200/admin"
 ```
```

Comment:
Not necessary to give the name of the program to start when it's `crate` and doesn't come with extra args.